src/registrar/views/domain.py

```diff
@@ -643,7 +643,46 @@ def _domain_abs_url(self):
         """Get an absolute URL for this domain."""
         return self.request.build_absolute_uri(reverse("domain", kwargs={"pk": self.object.id}))
 
-    def _make_invitation(self, email_address):
+    def _send_domain_invitation_email(self, email: str, add_success=True):
+        """Performs the sending of the domain invitation email,
+        does not make a domain information object
+        email: string- email to send to
+        add_success: bool- default True indicates:
+          adding a success message to the view if the email sending succeeds"""
+        # created a new invitation in the database, so send an email
+        domainInfoResults = DomainInformation.objects.filter(domain=self.object)
+        domainInfo = domainInfoResults.first()
```

Comment:
```suggestion
        domainInfoResults = DomainInformation.objects.filter(domain=self.object)
        # If/else logic here for 0 count - some kind of early return?
        # Can be .first() as well.
        domainInfo = domainInfoResults.get()
```
(Non-blocking/Nitpick) I don't think this will happen in normal flow but there is a risk here that "domainInfo" could be None, so an invalid email could be sent out. Ideally we should catch that or at least log it, rather than let the code itself define the fail condition, we can set one for it!

I also think that given that domain has a one to one relation with DomainInformation, why not use a get? By filtering on domain you will always get a return of either 0 or 1.